File: testprojects/master_python_mainone/pipelines/PYTHON_LIVY/code/python_livy/graph/Reformat_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from python_livy.config.ConfigStore import *
from python_livy.udfs.UDFs import *

def Reformat_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0

File: testprojects/scala_main/pipelines/SCALA_AZURE/code/src/main/scala/io/prophecy/pipelines/scala_azure/graph/Reformat_1.scala
package io.prophecy.pipelines.scala_azure.graph

import io.prophecy.libs._
import io.prophecy.pipelines.scala_azure.config.ConfigStore._
import io.prophecy.pipelines.scala_azure.udfs.UDFs._
import io.prophecy.pipelines.scala_azure.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Reformat_1 {
  def apply(spark: SparkSession, in: DataFrame): DataFrame = in
}